Clicker/ViewControllers/PollsViewController+Extension.swift

```diff
@@ -162,7 +162,10 @@ extension PollsViewController: SliderBarDelegate {
 extension PollsViewController: UITextFieldDelegate {
 
     func textFieldShouldReturn(_ textField: UITextField) -> Bool {
-        self.view.endEditing(true)
+        if let text = textField.text, text.count == 6 { // Valid code length
```

Comment:
Can we make `6` into a class variable at the top of `PollsViewController` like `let validCodeLength = 6`? Just in case we ever decide to change the valid code length.

Reply:
i'm gonna actually add it to `IntegerConstants` in case we use it anywhere else!